ext/ffi/tests/gh11934b.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Feature GH-11934 (Allow to pass CData into C variables)
+--EXTENSIONS--
+ffi
+zend_test
+--FILE--
+<?php
+require_once __DIR__ . '/utils.inc';
+$header = <<<HEADER
+extern int gh11934b_ffi_var_test_cdata;
+HEADER;
+
+if (PHP_OS_FAMILY !== 'Windows') {
+    $ffi = FFI::cdef($header);
+} else {
+    try {
+        $ffi = FFI::cdef($header, 'php_zend_test.dll');
+    } catch (FFI\Exception $ex) {
+        $ffi = FFI::cdef($header, ffi_get_php_dll_name());
+    }
+}
+
+var_dump($ffi->gh11934b_ffi_var_test_cdata);
+$source = $ffi->new('int');
+$source->cdata = 31;
+$ffi->gh11934b_ffi_var_test_cdata = $source;
+var_dump($ffi->gh11934b_ffi_var_test_cdata);
+
```

Comment:
```suggestion
$ffi->gh11934b_ffi_var_test_cdata = 2;

```

I guess we have to restore original value for repeated tests.

Reply:
Or not rely on the initial value at all, but always write our own initial value.